### ISSUE TITLE
uBlock breaks save page functionality

### ISSUE THREAD

**Issue by ghost** [gorhill/uBlock#1525]
Clean Chrome 49.0.2623.110 m installation, uBlock Origin v1.6.6 with default settings/filters.

Steps to reproduce:
1. Open any page with resources that will be blocked by uBlock.
2. Save page as "Webpage, Complete".

Expected result: successfully saved page.

Actual result: download stucks forever at 0-5%.

**Comment by gorhill** [comment]
Duplicate of #1185.
